packages/components/src/components/accordion-group/accordion-group.stories.ts

```diff
@@ -34,124 +39,49 @@ export const Default = {
+      <sd-accordion summary="Accordion 1">Sample content</sd-accordion
+      ><sd-accordion summary="Accordion 2">Sample content</sd-accordion
+      ><sd-accordion summary="Accordion 3">Sample content</sd-accordion>
+    </sd-accordion-group>
+  `
 };
 
-/**
- * Accordion group can be used with background options of white, neutral-100 and primary-100.
- */
-
-export const Samples = {
-  render: () => {
-    return html`
-      <div class="w-full flex gap-8 flex-col">
-        <div>
-          <div class="p-4 mb-8 bg-neutral-100 text-left text-[14px] font-bold box-border">white</div>
-
-          <sd-accordion-group class="w-1/2 bg-white p-8">
-            <sd-accordion summary="Accordion 1"
-              ><div class="slot slot--border slot--text h-16">Default slot</div></sd-accordion
-            ><sd-accordion summary="Accordion 2"
-              ><div class="slot slot--border slot--text h-16">Default slot</div></sd-accordion
-            ><sd-accordion summary="Accordion 3"
-              ><div class="slot slot--border slot--text h-16">Default slot</div></sd-accordion
-            >
-          </sd-accordion-group>
-        </div>
-
-        <div>
-          <div class="w-full p-4 mb-8 bg-neutral-100 text-left text-[14px] font-bold box-border">neutral-100</div>
-          <sd-accordion-group class="w-1/2 bg-neutral-100 p-8">
-            <sd-accordion summary="Accordion 1"
-              ><div class="slot slot--border slot--text h-16">Default slot</div></sd-accordion
-            ><sd-accordion summary="Accordion 2"
-              ><div class="slot slot--border slot--text h-16">Default slot</div></sd-accordion
-            ><sd-accordion summary="Accordion 3"
-              ><div class="slot slot--border slot--text h-16">Default slot</div></sd-accordion
-            >
-          </sd-accordion-group>
-        </div>
-
-        <div>
-          <div class="w-full p-4 mb-8 bg-neutral-100 text-left text-[14px] font-bold box-border">primary-100</div>
-          <sd-accordion-group class="w-1/2 bg-primary-100 p-8">
-            <sd-accordion summary="Accordion 1"
-              ><div class="slot slot--border slot--text h-16">Default slot</div></sd-accordion
-            ><sd-accordion summary="Accordion 2"
-              ><div class="slot slot--border slot--text h-16">Default slot</div></sd-accordion
-            ><sd-accordion summary="Accordion 3"
-              ><div class="slot slot--border slot--text h-16">Default slot</div></sd-accordion
-            >
-          </sd-accordion-group>
-        </div>
-      </div>
-    `;
-  }
+export const SampleWithPrimaryBackground = {
+  name: 'Sample with primary background',
+  render: () => html`
+    <sd-accordion-group class="bg-primary-100 p-8">
+      <sd-accordion summary="Accordion 1">Sample content</sd-accordion
+      ><sd-accordion summary="Accordion 2">Sample content</sd-accordion
+      ><sd-accordion summary="Accordion 3">Sample content</sd-accordion>
+    </sd-accordion-group>
+  `
```

Comment:
That's a weird one. @coraliefeil Why do we document this? This feels more like a UX hint, which could be done verbally but having this as a sample or explicitly visually documented feels... weird? (Especially: Why only for this, but not for others?)

Reply:
Moved to `templates` as discussed in the sync with @coraliefeil and @MartaPintoTeixeira.